.vib/spring-cloud-dataflow-composed-task-runner/goss/vars.yaml

```diff
@@ -0,0 +1,6 @@
+binaries:
+  - java
+directories:
+  - paths:
+    - /opt/bitnami/spring-cloud-dataflow-composed-task-runner
```

Comment:
We are not creating or configuring the permissions for this folder, as the application 'spring-cloud-dataflow-shell' does not have a postunpack.sh logic, could you please remove these lines?